tck/src/main/java/org/eclipse/microprofile/openapi/apps/beanvalidation/BeanValidationData.java

```diff
@@ -0,0 +1,259 @@
+/**
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.microprofile.openapi.apps.beanvalidation;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+import jakarta.validation.constraints.DecimalMax;
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.Negative;
+import jakarta.validation.constraints.NegativeOrZero;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.PositiveOrZero;
+import jakarta.validation.constraints.Size;
+import jakarta.validation.groups.Default;
+
+public class BeanValidationData {
+
+    @NotEmpty
+    private String notEmptyString;
+
+    @NotEmpty
+    private List<String> notEmptyList;
+
+    @NotEmpty
+    private Map<String, String> notEmptyMap;
+
+    @NotBlank
+    private String notBlankString;
+
+    @Size(min = 2, max = 7)
+    private String sizedString;
+
+    @Size(min = 1, max = 10)
+    private List<String> sizedList;
+
+    @Size(min = 3, max = 5)
+    private Map<String, String> sizedMap;
+
+    @DecimalMax("1.5")
+    private BigDecimal maxDecimalInclusive;
+
+    @DecimalMax(value = "1.5", inclusive = false)
+    private BigDecimal maxDecimalExclusive;
+
+    @DecimalMin("3.25")
+    private BigDecimal minDecimalInclusive;
+
+    @DecimalMin(value = "3.25", inclusive = false)
+    private BigDecimal minDecimalExclusive;
+
+    @Max(5)
+    private int maxInt;
+
+    @Min(7)
+    private int minInt;
+
+    @Negative
+    private int negativeInt;
+
+    @NegativeOrZero
+    private int negativeOrZeroInt;
+
+    @Positive
+    private int positiveInt;
+
+    @PositiveOrZero
+    private int positiveOrZeroInt;
```

Comment:
The intent with using the "exclusive" booleans with positive/negative wasn't immediately obvious to me due to the use of integers. Good catch.